themes/ddbasic/scripts/ddbasic.common.js

```diff
@@ -84,6 +84,17 @@
       $(this).toggleClass('js-toggled');
     });
 
+    $('.primary-content video, .primary-content audio').on('loadedmetadata', function (i) {
+      // If video size bigger then window size, it should be resized.
+      var video = i.currentTarget;
+      var width = $(this).parent().width() * 0.8;
+
+
+      if (video.videoWidth > width) {
+        $(this).width(width * 0.9);
+      }
+    });
```

Comment:
Why is this added to `ddbasic.common.js` instead of in `ding_dams`?

Also: Again 0.8 and 0.9 is used. Why is this?